scripts/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+line-length = 100
+skip-string-normalization = true
```

Comment:
I'm not an expert, but I didn't find where this file is used by next patching

Reply:
this file is read by the github action of `psf/black`. see https://github.com/psf/black/blob/bbfdba3a5e4f0b3105ec9526c51e769866b28f1f/action/main.py#L59C20-L59C34